README.md

```diff
@@ -50,7 +50,46 @@ You may need to run this command in order to get your `kubectl` command configur
 
 ##### To simplify deployment of containers on GCE you can push the image to `gcr.io`
 
-I'm following these [instructions](http://scottsmerchek.com/2015/07/24/pushing-to-google-container-registry-from-circleci/)
+Add some (below) environmental variables for each individual project/repo via the CircleCI UI. Click on the gear icon for a repo, then "Environmental Variables" under the ​*Tweaks*​ heading in the left-hand sidebar.
+
+First generate a new JSON key in the [developer console](https://console.developers.google.com/project/lexical-cider-93918/apiui/credential/serviceaccount/account-2@lexical-cider-93918.iam.gserviceaccount.com).
```

Comment:
And actually, I suggest adding the project id to the environment variables as I have done here: https://github.com/OAODEV/iadops-trafficking-service/commit/05847f3bfeb6d89d21aabad21aef5813e3f55f54